### ISSUE TITLE
Error Saving Bundle Meta Fields After Sorting List

### ISSUE THREAD

**Issue by cisnerosebastian** [gizburdt/cuztom#336]
This is What i've found, let me know if you see this problem. 

- You have a bundle. 
- You add one item.
- Add Another Item.
- Save post.
- So far, you should have 2 items.
- Then, add another item (the item is added to the end of the list as always).
- Move this item up to the top.
- Add another item.
- Move this last item to be second item.
- Save.
PROBLEM!

You will notice that the first item is not saved. Just the lastest one.

the problem happens when you add an item, move it, and then add more items. Only the latest item added is Saved.

Do you see the problem?

**Comment by gizburdt** [comment]
Hi, thanks for noticing :) I will have a look at it soon.

**Comment by cisnerosebastian** [comment]
Hi gizburdt, I just installed the 2.9.16 version of your class, and you CAN'T do what i told you (the problem). You can't drag and drop the latest added item. The problem I've explained was in versio 2.9.4.

The thing that you can't move the latest added item in a bundle is kind of anoying, but it does not leave you go into the bug I explained before.

**Comment by MarGul** [comment]
Noticing the same problem. The reason why is because when you add a new item. The javascript will take a look at your last item, grab that ID and then increment it by 1 for you new box. Because you have moved around your items maybe the new item will have the same ID as another box and therefore overwrite it.

This is the fix that I came up with, working fine for me. in functions.js on line 187 and 190, dont forget to remove the variables last and new_item that gets set here. Or just overwrite them with the new way.

var last;
var lastID = 0;
$('.js-cuztom-sortable-item').each(function() {
	// For each of your items, look at the ID of this item
         var id = parseInt($(this).find('.cuztom-select').attr('name').match(/\[(\d+)\]/)[1]);
		// If the ID of this item is greater then any other of the items, update the ID and the last item
               if(id > lastID) {
			lastID = id;
			last   = $(this);
		}
	});
// Clone the item that we found with the highest ID
var new_item = last.clone( false, false );
